site/src/docs/components/login/components.mdx

```diff
@@ -0,0 +1,205 @@
+  <p>Hello!</p>
+</LoginProvider>;
+```
+
+</PlaygroundPreview>
+
+### LoginButton
+
+The login button handles the redirection to the OIDC server and also errors if the server rejects the request for <ExternalLink href="https://tunnistamo.dev.hel.ninja/.well-known/openid-configuration">openID configuration</ExternalLink>.
+
+### Properties
+
+| Property                                           | Description                                                       | Default            |
+| -------------------------------------------------- | ----------------------------------------------------------------- | ------------------ |
+| `errorText`                                        | Required. Error text shown, if OIDC server rejects the connection | none               |
+| `spinnerColor`                                     | Optional. Colour of the loading spinner.                          | var(--color-white) |
+| [Table 2: Properties of the LoginButton component] |
+
+Button text passed and rendered as a child.
+
+<PlaygroundPreview>
+
+```jsx
+<LoginButton errorText="Login failed. Try again!">Log in</LoginButton>
+```
+
+</PlaygroundPreview>
+
+### LoginCallbackHandler
+
+When the OIDC server redirects the browser back to the given callback route, this component handles the response.
+
+### Properties
+
+| Property                                                    | Description                                                                                                          |
+| ----------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| `onSuccess`                                                 | Required. The function is called when a response is handled successfully. The function is called with a User object. |
+| `onError`                                                   | Required. The function is called when a login fails. The function is called with an OidcClientError.                 |
+| `children`                                                  | Optional. Usually child elements to inform the user that a response is being processed.                              |
+| [Table 3: Properties of the LoginCallbackHandler component] |
+
+<PlaygroundPreview>
+
+```jsx
+const onSuccess = (user: User) => {
+  // handle success
+  // Usually redirect to a page
+};
+const onError = (error: OidcClientError) => {
+  // handle error
+};
+return (
+  <LoginProvider {...providerProperties}>
+    <LoginCallbackHandler onSuccess={onSuccess} onError={onError}>
+      <div>Logging in...</div>
+    </LoginCallbackHandler>
+  </LoginProvider>
+);
+```
+
+</PlaygroundPreview>
+
+### SessionEndedHandler
+
+This component listens to the session ended signal. It is emitted from the Session poller, if it receives a response indicating the user's session has ended.
+Then the component shows a dialog and forces the user to logout.
```

Comment:
![image](https://github.com/City-of-Helsinki/helsinki-design-system/assets/2639230/d7b043e3-95cf-4908-8d4a-e08f7ba6def3)

It's a basic HDS Dialog. I added that information to the docs and link to storybook where it can be seen.

Reply:
Added info where the component should be positioned. Anywhere as long as it is under LoginProvider